package.json

```diff
@@ -49,6 +49,7 @@
     "q-io": "^2.0.3",
     "saucelabs": "^0.1.1",
     "wd": "^0.2.21",
-    "weak-map": "^1.0.5"
+    "weak-map": "^1.0.5",
+    "benchmark": "^1.0.0"
```

Comment:
You can add a `"scripts.benchmarks": "node benchmarks"` so we can do `npm run benchmarks`